DevelopGitHub/GitHubExtension/GitHubExtension.LocalizationTool/ViewModel/MainWindowViewModel.cs

```diff
@@ -0,0 +1,257 @@
+                "Error - Localization Tool",
+                MessageBoxButton.OK,
+                MessageBoxImage.Error);
+        }
+
+        public static void ShowInformationMessageBox(string information)
+        {
+            MessageBox.Show(information, "OK - Localization Tool", MessageBoxButton.OK, MessageBoxImage.Information);
+        }
+
+        private DelegateCommand addEmptyRowCommand;
+
+        public ICommand AddEmptyRowCommand
+        {
+            get
+            {
+                if (addEmptyRowCommand == null)
+                {
+                    addEmptyRowCommand = new DelegateCommand(AddEmptyRow);
+                }
+                return addEmptyRowCommand;
+            }
+        }
+
+        private void AddEmptyRow(object param)
+        {
+            TranslationData.Add(new TranslationDataRow());
+        }
+
+        private DelegateCommand saveJsonCommand;
+
+        public ICommand SaveJsonCommand
+        {
+            get
+            {
+                if (saveJsonCommand == null)
+                {
+                    saveJsonCommand = new DelegateCommand(SaveJson);
+                }
+                return saveJsonCommand;
+            }
+        }
+
+        private void SaveJson(object param)
+        {
+            JsonHelper.RemoveEmptyRows();
+            foreach (Lang value in Enum.GetValues(typeof(Lang)))
+            {
+                File.WriteAllText(Translator.GetFileName(value), JsonHelper.GenerateJson(value));
+            }
+
+            ShowInformationMessageBox("Saved!");
+        }
+
+        private DelegateCommand openJsonCommand;
+
+        public ICommand OpenJsonCommand
+        {
+            get
+            {
+                if (openJsonCommand == null)
+                {
+                    openJsonCommand = new DelegateCommand(OpenJson);
+                }
+                return openJsonCommand;
+            }
+        }
+
+        private void OpenJson(object param)
+        {
+            TranslationData.Clear();
+            foreach (Lang value in Enum.GetValues(typeof(Lang)))
+            {
+                JsonHelper.ReadJsonFromFile(value);
+            }
+
+            JsonHelper.RemoveEmptyRows();
+            ShowInformationMessageBox("Oppened!");
+        }
+
+        private DelegateCommand clearTranslationDataCommand;
+
+        public ICommand ClearTranslationDataCommand
+        {
+            get
+            {
+                if (clearTranslationDataCommand == null)
+                {
+                    clearTranslationDataCommand = new DelegateCommand(ClearTranslationData);
+                }
+                return clearTranslationDataCommand;
+            }
+        }
+
+        private void ClearTranslationData(object param)
+        {
+            TranslationData.Clear();
+        }
+
+        private DelegateCommand translatingButtonCommand;
+
+        public ICommand TranslatingButtonCommand
```

Comment:
Can you initialize all your command inside constructor of ViewModel using lambda syntax?